releasenotes/notes/custom-gradients-35923fa70af97d8d.yaml

```diff
@@ -0,0 +1,20 @@
+---
+
+fixes:
+  - |
+    This release note solves two issues : 
+    - Previously in the :class:`~qiskit_machine_learning.algorithms.QuantumGenerator` of 
```

Comment:
```suggestion
    This release note solves two issues : 

    - Previously in the :class:`~qiskit_machine_learning.algorithms.QuantumGenerator` of 
```
There needs to be blank line preceding any list definition for Sphinx otherwise all the text is wrapped up as a single paragraph as you can see from this output
![image](https://user-images.githubusercontent.com/40241007/170352060-070ac775-d410-4696-9cdb-56038976e4f4.png)